types/tx_msg.go

```diff
@@ -5,17 +5,19 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/cosmos/gogoproto/proto"
 	protov2 "google.golang.org/protobuf/proto"
 
+	coretransaction "cosmossdk.io/core/transaction"
+
 	"github.com/cosmos/cosmos-sdk/codec"
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 )
 
 type (
 	// Msg defines the interface a transaction message needed to fulfill.
-	Msg = proto.Message
+	// DEPRECATED: use cosmossdk.io/core/transaction.Type
```

Comment:
aren't we already using ` cosmossdk.io/core/transaction.Type`, why still DEPRECATED note?

Reply:
forcing function to get people to use core mainly. Happy to remove it

Reply:
would be nice to remove to avoid confusions